chore: add CONTRIBUTING.md file (#466)
keptn/lifecycle-toolkit: CONTRIBUTING.md

> # Contributing

Reviewer: [nitpick]: pls get rid of the `<br>`eaks in favor of normal newlines or enumerations/lists :)

## Linters

> This project uses a set of linters to ensure a good code quality.

Reviewer: ```suggestion
This project uses a set of linters to ensure good code quality.
```

> In order to make proper use of those linters inside an IDE, the following configurations are required.<br>

Reviewer: ```suggestion
In order to make proper use of those linters inside an IDE, the following configuration is required.<br>
```

> Further information can also be found at the `golangci-lint` documentation [here](https://golangci-lint.run/usage/integrations/)

Reviewer: ```suggestion
Further information can also be found in the [`golangci-lint` documentation](https://golangci-lint.run/usage/integrations/).
```

### Visual Studio Code

In Visual Studio Code the [Golang](https://marketplace.visualstudio.com/items?itemName=aldijav.golangwithdidi) extension is required.<br>
Adding the following lines to the `Golang` extension configuration file will enable all linters used in this project.

```
"go.lintTool": {
	"type": "string",
	"default": "golangci-lint",
	"description": "GolangGCI Linter",
	"scope": "resource",
	"enum": [
		"golangci-lint",
	]
},
"go.lintFlags": {
	"type": "array",
	"items": {
		"type": "string"
	},
	"default": ["--fast", "--fix"],
	"description": "Flags to pass to GCI Linter",
	"scope": "resource"
},
```

### GoLand / IntelliJ

In GoLand or IntelliJ, the plugin [Go Linter](https://plugins.jetbrains.com/plugin/12496-go-linter) will be required.<br>

> The plugin can bi installed via `Settings` >> `Plugins` >> `Marketplace`, search for `Go Linter` and install it.<br>

Reviewer 2: ```suggestion
The plugin can be installed via `Settings` >> `Plugins` >> `Marketplace`, search for `Go Linter` and install it.<br>
```
